testui/support/logger.py

```diff
@@ -145,19 +146,13 @@ def __file_log(log_file="stdout.log"):
     :param log_file: String
     :return: String
     """
-    root_dir = (
-        os.path.dirname(
-            os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-        )
-        + "/"
-    )
-    Path(root_dir + "appium_logs").mkdir(parents=True, exist_ok=True)
+    Path(ROOT_DIR).mkdir(parents=True, exist_ok=True)
     file_name: str
     if log_file == "stdout.log":
-        file_name = f"appium_logs/TEST_UI-{log_file}"
+        file_name = os.path.join(ROOT_DIR, f"TEST_UI-{log_file}")
```

Comment:
that would be quite challeging, since each time it writes a log, it writes it in that file, and that means it would have to reference to the time the file was created first time. so the time shoud be stored or passed each time the log method is used...